sample.env

```diff
@@ -1,12 +1,15 @@
-MONGO_DB_URI=Xx
-
-PORT=4000
-
-CLUSTER=yes
-
-BASE_URL=http://localhost:4000
-FRONTEND_URL=http://localhost:3000
-TOKEN_SECRET=XX
-
-GITHUB_CLIENT_ID=XX
-GITHUB_CLIENT_SECRET=XX
+MONGO_DB_URI=XX
+
+PORT=3000
+
+CLUSTER=yes
+
+BASE_URL=http://localhost:3000
```

Comment:
```
PORT=4000

CLUSTER=yes

BASE_URL=http://localhost:4000
FRONTEND_URL=http://localhost:3000
```

Make default backend to port `4000` and frontend to `3000`

Reply:
Done